src/components/bluebutton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, label, ...props }) =>
+(
+  <div
+    className={styles.our_button}
+    role="button"
+    tabIndex="-1"
+    {...props}
+    onClick={(event) => {
+      event.preventDefault();
```

Comment:
I read [the manual](https://facebook.github.io/react/docs/handling-events.html) and came to the conclusion that `onClick={onClick]` is *wrong* (the `this` will be rebound, leading to weird bugs).

`event.preventDefault()` is not necessary here, because a plain button does not have a default.

So suggest `onClick={(e)=>onClick(e)}`. Like the last example in the manual.